versioned_docs/version-v1.28.x/getting-started/release-notes/v1_28_5.md

```diff
@@ -39,27 +18,54 @@ Zowe Version 1.28.5 contains the bug fixes that are described in the following t
 
 #### Zowe CLI (Core)
 
-#### Zowe CLI Imperative Framework
+- Updated Imperative to version `4.18.19` to fix issues with normalizing newlines on file uploads. ([#1815](https://github.com/zowe/zowe-cli/issues/1815))
+
+- Updated Imperative to version `4.18.18` to use `@zowe/secrets-for-zowe-sdk` as the default credential manager.
 
-#### CICS Plug-in for Zowe CLI
+- Updated Imperative to include bugfixes in version `4.18.17`.
 
-#### DB2 Plug-in for Zowe CLI
+#### Zowe CLI Imperative Framework
 
-#### MQ Plug-in for Zowe CLI
+- Replaced use of `node-keytar` with the new `keyring` module from `@zowe/secrets-for-zowe-sdk`. ([zowe-cli#1622](https://github.com/zowe/zowe-cli/issues/1622))
 
-#### IMS Plug-in for Zowe CLI
+- Updated semver transitive dependency for technical currency.
 
 ### Zowe Explorer
 
+- Replaced `keytar` dependency with `keyring` module from [`@zowe/secrets-for-zowe-sdk`](https://github.com/zowe/zowe-cli/tree/master/packages/secrets). ([#2358](https://github.com/zowe/vscode-extension-for-zowe/issues/2358))
+
 ### Vulnerabilities fixed
 
 Zowe discloses fixed vulnerabilities in a timely manner giving you sufficient time to plan your upgrades. Zowe does not disclose the vulnerabilities fixed in the latest release as we respect the need for at least 45 days to decide when and how you upgrade Zowe. When a new release is published, Zowe publishes the vulnerabilities fixed in the previous release. For more information about the Zowe security policy, see the [Security page](https://www.zowe.org/security.html) on the Zowe website.
 
 The following security issues were fixed by the Zowe security group in version 1.27.x.
```

Comment:
This should probably be updated to say "... version 1.28.5.

Reply:
This list includes fixes from the _last_ version. I changed this to 1.28.4.